integration/simulation/simulation.go

```diff
@@ -56,9 +60,10 @@ func (s *Simulation) Start() {
 	// Arbitrary sleep to wait for RPC clients to get up and running
 	// and for all l2 nodes to receive the genesis l2 batch
 	// todo - instead of sleeping, it would be better to poll
-	time.Sleep(10 * time.Second)
+	//time.Sleep(10 * time.Second) //todo - determine if this is till necessary?
```

Comment:
did you see any extra flakyness?

Reply:
I think I did, yes. Will revert this, tho I really do not like it.